molo/globalsite/tests/test_views.py

```diff
@@ -20,7 +20,7 @@ def setUp(self):
         asia = Region.objects.create(name='Asia')
 
         CountrySite.objects.create(
-            name='South Africa', code='ZA',
+            name='South Africa', code='za',
```

Comment:
The the test for both lower case as upper country code?

Reply:
@Mitso the next country I'm creating it has the locale with upper country code